### PR TITLE
Update einmaleins.tex

### PR DESCRIPTION
**Pull request by JoJoBarthold2**
sehr wichtiges adden der besten kogni memeseite

### CODE REVIEW

**Review by PhictionalOne — COMMENTED** [review]
@linususer @dakopen Is this okay for you? Does the FSK support this? Also do you want to include `fsk.tuebingen` Instagram as well?

File: kogni/erste_woche/einmaleins.tex [hunk 1]
```diff
@@ -48,6 +48,8 @@
 
 \item[Mensa:] Die Mensen sind der Ort in dem man sich zum (un-)regelmäßigen Mittagessen trifft. Jede Mensa hat einen Speiseplan, welchen man auf der Seite des Studierendenwerks einsehen kann\footnote{\url{http://www.my-stuwe.de/mensa/}}.	%TODO insert \link{}{}?
 
+\item[Memes] Eine Memeseite, bei der jeder mitmachen kann ist auf Instagram unter kogni_memes zu finden
```
Comment (PhictionalOne) [inline kogni/erste_woche/einmaleins.tex:51]: Please add colon after `Memes`.

Also use `\href{https://www.instagram.com/kogni_memes/}{\faInstagram @kogni\_memes}` to get a more nice render.

**Review by PhictionalOne — CHANGES_REQUESTED** [review]

File: kogni/erste_woche/einmaleins.tex (hunk 1 above)
Comment (JoJoBarthold2) [reply kogni/erste_woche/einmaleins.tex:51]: okay. btw idk if FSK supports this but they should. In any case i can make a meme out of this

Comment (linususer) [reply kogni/erste_woche/einmaleins.tex]: @PhictionalOne  we will discuss this at our session and I will come back here

**Comment by PhictionalOne** [comment]
Honestly e9ad67c 's Commit message is really bad... At least I can squash it.

**Pull request closed without merging**